contracts/strategies/LPOptimiserStrategy.sol

```diff
@@ -154,26 +124,27 @@ contract LPOptimiserStrategy is DefaultAccessControl {
 
     /// @notice Set new optimimal tick range based on current tick
     /// @param currentFixedRateWad currentFixedRate which is passed in from a 7-day rolling avg. historical fixed rate.
-    function rebalance(uint256 currentFixedRateWad) public returns (int24 newTickLowerMul, int24 newTickUpperMul) {
+    /// @return newTickLowerMul The new lower tick for the rebalanced position
+    /// @return newTickUpperMul The new upper tick for the rebalanced position
+    function rebalanceTicks(uint256 currentFixedRateWad) public returns (int24 newTickLowerMul, int24 newTickUpperMul) {
         _requireAtLeastOperator();
 
         // 0. Get tickspacing from vamm
-        // _tickSpacing = _vamm.tickSpacing(_vamm);
+        int24 tickSpacing = _vamm.tickSpacing();
 
         // 1. Get the new tick lower
-        // write UTs to check for underflow
         int256 deltaWad = int256(currentFixedRateWad) - int256(_sigmaWad);
-        int256 newFixedLowerWad = 0;
-        if (deltaWad > 0) {
-            // delta is greater than 0 => choose delta
-            if (deltaWad < _max_possible_lower_bound_wad) {
+        int256 newFixedLowerWad;
+        if (deltaWad > 1e15) {
```

Comment:
i'd put 1e15 into a constant at the top of the contract, the constant can be named "MINIMUM_FIXED_RATE"

Reply:
given 1e15 is right on the edge, is it worth making it e.g. 1e16 instead, i.e. 0.01 @0xZenus @Sol3iman ?

Reply:
Agree. 1e16 would be better.